MIVisionX-setup.py

```diff
@@ -185,6 +185,8 @@
 	os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y '+linuxSystemInstall_check+' install qt5-default qtcreator')
 	# Install RPP
 	if rppInstall == 'yes':
+		os.system('(sudo apt-get install libjsoncpp-dev)')
+		os.system('(sudo apt --yes install libboost-all-dev clang)')
```

Comment:
apt/apt-get does not work on CentOS, to use the install pattern in line 185

Reply:
@LakshmiKumar23 any updates on this PR?

Reply:
Updated for cent-os. Tested on both platforms.